cmd/kubeadm/BUILD

```diff
@@ -5,7 +5,7 @@ load(
     "go_binary",
     "go_library",
 )
-load("//vendor/k8s.io/client-go/pkg/version:def.bzl", "version_x_defs")
```

Comment:
Let's see what the actual problem is, instead of reverting straight to this.
/hold